imdb/views.py

```diff
@@ -0,0 +1,56 @@
+import json
+import pdb
+import sys
+import traceback
+
+import requests
+
+import SearchResults
+
+
+from bs4 import BeautifulSoup
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.core.paginator import Paginator, PageNotAnInteger, EmptyPage
+
+
+
+def search_movie(request):
+
+    movie = request.GET.get('q','')
+    error, status, movie_list = SearchResults.get_search_movies(movie)
+
+    if error:
+        data = json.dumps(movie_list)
+        return HttpResponse(data,
+                            content_type='application/json',
+                            status=status)
+    paginator = Paginator(movie_list, 25)
```

Comment:
correct
first param is a python serializable(list in this case) and second is the page size